Velaptor/PublicExtensionMethods.cs

```diff
@@ -261,5 +288,280 @@ public static GlyphMetrics ApplySize(this GlyphMetrics value, float size)
+            }
+
+            // Must have a file name with an extension
+            return !string.IsNullOrEmpty(Path.GetFileName(path)) && !string.IsNullOrEmpty(Path.GetExtension(path));
+        }
+
+        /// <summary>
+        /// Returns a value indicating if the given <see langword="string"/> <paramref name="path"/>
+        /// is an invalid fully qualified path.
+        /// </summary>
+        /// <param name="path">The path to check.</param>
+        /// <returns><c>true</c> if invalid.</returns>
+        /// <example>
+        /// <list type="number">
+        ///     <item>Value of <c>null</c> will return <c>true</c>.</item>
+        ///     <item><c>null</c> or <c>empty</c> will returns <c>true</c>.</item>
+        ///     <item>
+        ///         Start with a drive letter
+        ///         <para>Value of <c>C:/</c> will return <c>false</c>.</para>
+        ///     </item>
+        ///     <item>
+        ///         Contain at least 1 directory
+        ///         <para>will return <c>false</c>: C:/my-directory</para>
+        ///     </item>
+        ///     <item>
+        ///         Must contain a file name with file extension
+        ///         <list type="bullet">
+        ///             <item>Value of <c>C:/my-directory/my-file.txt</c> will return <c>false</c>.</item>
+        ///             <item>Value of <c>C:/my-directory/my-file</c> will return <c>true</c></item>
+        ///             <item>Value of <c>C:/my-directory/.txt</c> will return <c>true</c></item>
+        ///         </list>
+        ///     </item>
+        /// </list>
+        /// </example>
+        /// <remarks>
+        ///     This does not check if the path actually exists.  This is for valid path syntax checks only.
+        /// </remarks>
+        public static bool IsInvalidFilePath(this string path) => !IsValidFilePath(path);
+
+        /// <summary>
+        /// Returns a value indicating if the given <paramref name="path"/> a valid drive.
+        /// </summary>
+        /// <param name="path">The path to check.</param>
+        /// <returns>True if <paramref name="path"/> contains a valid drive.</returns>
+        /// <example>
+        /// <list type="number">
+        ///     <item>Value of <c>C:/my-dir</c> will return <c>true</c>.</item>
+        ///     <item>Value of <c>C:my-dir</c> will return <c>true</c>.</item>
+        ///     <item>Value of <c>C/my-dir/my-file.txt</c> will return <c>false</c>.</item>
+        ///     <item>Value of <c>//my-dir</c> will return <c>false</c>.</item>
+        ///     <item>Value of <c>./my-dir</c> will return <c>false</c>.</item>
+        ///     <item>Value of <c>../my-dir</c> will return <c>false</c>.</item>
+        /// </list>
+        /// </example>
+        /// <remarks>
+        ///     This does not check if the drive actually exists.  This is for valid path syntax checks only.
+        /// </remarks>
+        public static bool ContainsValidDrive(this string path)
```

Comment:
Refactor name to below and adjust usages and XML code comments as necessary

```suggestion
        public static bool HasValidDriveSytax(this string path)
```